### DISCUSSION TITLE
Parsing Remark Field

### DISCUSSION THREAD

**Discussion by ozhangulen** [mivek/MetarParser#591] [Q&A]
MetarParser is a nice tool to parse metar data. But i have a problem with it. My metar data contains remark data in it. MetarParser parses the data I provide and give me a result like this in its result:

Remark: RWY21 02003KT 350V060

 Is there any utility to parse remark info in more detail.

Best regards...

**Comment by mivek** [comment]
Hello @ozhangulen 

Can you please provide the full metar?
Due to the complexity of the remark section, the representation is a single string. It is parsed using the [RemarkParser](https://github.com/mivek/MetarParser/blob/master/metarParser-parsers/src/main/java/io/github/mivek/parser/RemarkParser.java). The remark parser does not use the same parse the same elements than the MetarParser and the TAFParser.
